### PR TITLE
Initialization of the game after receiving START_GAME_RESPONSE

### PR DESCRIPTION
**Pull request by v4m3rrr**
Solved #60 
Implemented StartGameController class and fixed issues with joining to room.

### CODE REVIEW

File: app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java [hunk 1]
```diff
@@ -6,55 +6,168 @@
...
+                        noble.sapphireMinesRequired,
+                        noble.rubyMinesRequired,
+                        noble.diamondMinesRequired,
+                        noble.onyxMinesRequired
+                ));
+            }
 
-            // TODO Initialize the entire model
-            // TODO Move user to the GameActivity
+            List<TokenType> tokenTypes = new ArrayList<>(EnumSet.allOf(TokenType.class));
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java]: What is it for?

File: app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java [hunk 2]
```diff
@@ -6,55 +6,168 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java]: -_-

**Review by kestivvi — APPROVED** [review]
Those conversions from type to type are awesome 😎

**Review by kestivvi — CHANGES_REQUESTED** [review]
Please delete not needed empty lines and not used variables

File: app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java (hunk 1 above)
Comment (v4m3rrr) [reply app/src/main/java/com/example/splendormobilegame/activities/WaitingRoom/StartGameController.java]: Good question

**Review by kestivvi — APPROVED** [review]
🎂

**Pull request merged**